src/storage/mvcc/reader/scanner/backward.rs

```diff
@@ -266,6 +265,18 @@ impl<S: Snapshot> BackwardKvScanner<S> {
                     if self.met_newer_ts_data == NewerTsCheckState::NotMetYet {
                         self.met_newer_ts_data = NewerTsCheckState::Met;
                     }
+                    if self.cfg.isolation_level == IsolationLevel::RcCheckTs {
```

Comment:
Does the self.cfg.isolation_level come from tidb or tikv's config file?

Reply:
No it comes from the rpc requst field `isolation_level` which is set by read requests in https://github.com/pingcap/tidb/pull/32922.